keyboards/handwired/ortho5x14/keymaps/2u/keymap.c

```diff
@@ -0,0 +1,597 @@
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  //print("hello keypress.\n");
+  #ifdef CONSOLE_ENABLE
+    //uprintf("%s keycode\n", keycode);
+    uprintf("process --> KL: kc: %u, col: %u, row: %u, pressed: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed);
+  #endif
+
+  switch (keycode) {
+    case KC_TRNS:
+    case KC_NO:
+      /* Always cancel one-shot layer when another key gets pressed */
+      if (record->event.pressed && is_oneshot_layer_active())
+      clear_oneshot_layer_state(ONESHOT_OTHER_KEY_PRESSED);
+      return true;
+    case RESET:
+      /* Don't allow reset from oneshot layer state */
+      if (record->event.pressed && is_oneshot_layer_active()){
+        clear_oneshot_layer_state(ONESHOT_OTHER_KEY_PRESSED);
+        return false;
+      }
+      return true;
+    case QWERTY:
+      if (record->event.pressed) {
+        persistent_default_layer_set(1UL<<_QWERTY);
+      }
+      return false;
+      break;
+    case LOWER:
+      if (record->event.pressed) {
+        layer_on(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      } else {
+        layer_off(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      }
+      return false;
+      break;
+    case RAISE:
+      if (record->event.pressed) {
+        layer_on(_RAISE);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      } else {
+        layer_off(_RAISE);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      }
+      return false;
+      break;
+    case MOUSE:
+      if (record->event.pressed) {
+        persistent_default_layer_set(1UL<<_MOUSE);
+      }
+      return false;
+      break;
+  }
+  return true;
+};
+
+void post_process_record_user(uint16_t keycode, keyrecord_t *record) {
+  #ifdef CONSOLE_ENABLE
+    //print("Running post_process_record_user.\n");
+    uprintf("post_process --> KL: kc: %u, col: %u, row: %u, pressed: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed);
+  #endif
+  if (!record->event.pressed) {
+    if(keyboard_report->mods & MOD_BIT(KC_LCTL)) {
+      #ifdef CONSOLE_ENABLE
+        print("FOUND 2nd challenge:  KC_LCTL is enabled.\n");
+      #endif
+      if(keyboard_report->mods & MOD_BIT(KC_LSFT)) {
+        #ifdef CONSOLE_ENABLE
+          print("FOUND 3rd challenge:  KC_LSFT is enabled.\n");
+        #endif
+        // Remove Left Ctrl and Left Shift keys from Triple Tap of Left Ctrl key.
+        // Works with 'ctl_finished' function, Triple Tap.
+        unregister_mods(MOD_BIT(KC_LCTL));
+        unregister_mods(MOD_BIT(KC_LSFT));
+        #ifdef CONSOLE_ENABLE
+          print("** MODs for LCTL and LSFT have been released.\n");
+        #endif
+      }
+    }
+  }
+}
+
+void matrix_init_user(void) {
+    #ifdef AUDIO_ENABLE
+        startup_user();
+    #endif
+}
+
```

Comment:
this isn't needed as it's called by default. 
```suggestion

Reply:
Done.